server/src/main/java/org/elasticsearch/common/blobstore/BlobContainer.java

```diff
@@ -120,9 +120,7 @@ default void deleteBlobsIgnoringIfNotExists(List<String> blobNames) throws IOExc
         IOException ioe = null;
         for (String blobName : blobNames) {
             try {
-                deleteBlob(blobName);
```

Comment:
Just saw this and figured I'd dry this up here, no need to have duplicate logic for suppressing that exception.